### ISSUE TITLE
idea about tracking individual progress via google account

### ISSUE THREAD

**Issue by mitramartin** [andreimoment/tangomanual#71]
hi

love the idea to have a link where people can track their mission completes via logging into google account

someday, the info could be connected to the mission pairing algorithm so the host can always see where each student is

let me know how i could help this happen

**Comment by andreimoment** [comment]
what is the mission pairing algorithm in human language?

**Comment by mitramartin** [comment]
it is an algorithm, in a google sheets macro, that figures out the most constructive way to pair people up during a lab to teach/learn missions

**Comment by andreimoment** [comment]
In what way? 

Andrei

**Comment by mitramartin** [comment]
emailed you the file, can you see how it does it?

**Comment by andreimoment** [comment]
No, I can't figure it out. Can you describe in human language what it does? 

Andrei

**Comment by mitramartin** [comment]
essentially it attempts to create constructive pairings using the following parameters: 

1) beginners (level 1) learn 100% of the time
2) host teacher teaches 100% of the time
3) everyone else learns 50% of the time
4) missions must be explored sequentially
5) it is ok to have 1:1, 1:2 or 1:3 pairings
6) any intermediates newly joining need to start at the beginning (level 1)
7) if you are late, you cannot join until the next shift
8) peers decide together if/when they feel the person could teach that mission and thus is ready to move on

**Comment by andreimoment** [comment]
Excellent!

Does the facilitator make decisions overriding this logic and in what situations?

I think the 1:2 and 1:3 pairings may create a bit of complexity. When would they come into place?

**Comment by mitramartin** [comment]
yes facilitator can override

1:2 or 1:3 is only if that is the only way to solve situation

**Comment by andreimoment** [comment]
For 4., I noticed that this may not always be the case. For example, I didn't go through level 1 missions before teaching them. It would've been nice if I did, but it's also nice to learn things on my level which are immediately interesting and challenging. Doing this however means that I am not qualified to teach the level 1 missions.
